infra/nnfw/CMakeLists.txt

```diff
@@ -56,13 +56,12 @@ macro(nnas_find_package PREFIX)
   )
 endmacro(nnas_find_package)
 
-# C++14 feature requires 5 or later
-# Using std::unordered_map shows build fail under 6.2
-if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.2)
-  message(FATAL "Runtime build requires GNU Compiler version 6.2 or later.")
+# C++17 feature requires 9.1 or later for stable usage
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.1)
+  message(FATAL "Runtime build requires GNU Compiler version 9.1 or later.")
```

Comment:
https://gcc.gnu.org/onlinedocs/libstdc++/manual/status.html#status.iso.2017

>GCC 9.1 was the first release with non-experimental C++17 support, so the API and ABI of features added in C++17 is only stable since that release.